auth/lib/ui/settings/language_picker.dart

```diff
@@ -181,8 +181,6 @@ class _ItemsWidgetState extends State<ItemsWidget> {
         return '日本語';
       case 'ko':
         return '한국어';
-      case 'ar':
-        return 'العربية';
```

Comment:
Wondering if this is supposed to be a part of this change?

Reply:
This is a part of lint changes, if you see 'ar' is repeated key, it is the first case also.

Reply:
See: https://github.com/ente-io/ente/blob/83990e11601e52caf01dadc6cb31aad26047c705/auth/lib/ui/settings/language_picker.dart#L127-L128